cli/api/shell.go

```diff
@@ -238,7 +238,9 @@ func (a *api) RunShell(config ShellConfig) error {
 		// Delete the container
 
 		if err := dockercli.StopContainer(container.ID, 10); err != nil {
-			glog.Fatalf("failed to stop container: %s (%s)", container.ID, err)
+			if _, ok := err.(*dockerclient.ContainerNotRunning); !ok {
+			        glog.Fatalf("failed to stop container: %s (%s)", container.ID, err)
+			}
 		} else if err := dockercli.RemoveContainer(dockerclient.RemoveContainerOptions{ID: container.ID}); err != nil {
```

Comment:
This won't work, you need to do something like
dockercli.StopContainer
if err := dockercli.RemoveContainer(); err != nil {
   glog.Errorf
}